test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -192,6 +261,1427 @@ public void testCreateDeleteKeyValueTable() {
+                this.controllerService.createKeyValueTable(ModelHelper.decode(newScope, newKVT.getKeyValueTableName(),
+                        configList.get(index)), observerList.get(index));
+                return observerList.get(index);
+            }, requestPool);
+            requestFutures.thenApply(x -> resultMap.put(index, x.get().getStatus()));
+            requestFutures.thenRun(() -> log.info("Thread request: {}", index));
+            requestList.add(requestFutures);
+        }
+
+        //wait for create Table requests completion
+        Futures.allOf(requestList).get();
+        ExecutorServiceHelpers.shutdown(this.requestPool);
+
+        log.info("result map: {}", resultMap);
+        int partitionSize = this.controller.getCurrentSegmentsForKeyValueTable(newScope,
+                newKVT.getKeyValueTableName()).join().getSegments().size();
+        log.info("Partition size of created table: {}", partitionSize);
+
+        resultMap.forEach((k, v) -> {
+            if (v.equals(CreateKeyValueTableStatus.Status.SUCCESS)) {
+                assertEquals("The corresponding table should only be created against which the creation " +
+                                "request is accepted", configList.get(k).getPartitionCount(), partitionSize);
+                successCount.incrementAndGet();
+            } else if (v.equals(CreateKeyValueTableStatus.Status.TABLE_EXISTS)) {
+                assertNotEquals("The corresponding table should NOT be created against which the same name " +
+                                "table already exists", configList.get(k).getPartitionCount(), partitionSize);
+                tableExistsCount.incrementAndGet();
+            } else {
+                throw new ControllerFailureException("Unknown return status creating table" + v);
+            }
+        });
+
+        assertEquals("Only one table creation request must return success status", 1,
+                successCount.intValue());
+        assertEquals("Other two table creation request must return table exists status ", 2,
+                tableExistsCount.intValue());
+        Iterator<KeyValueTableInfo> tableItr = this.controller.listKeyValueTables(newScope).asIterator();
+        List<KeyValueTableInfo> tableList = new ArrayList<>();
+        while (tableItr.hasNext()) {
+            tableList.add(tableItr.next());
+        }
+        log.info("Table List: {}", tableList);
+        assertEquals("Only one table should be created", 1, tableList.size());
+        assertEquals("The same created table should be present in the table list",
+                newKVT.getKeyValueTableName(), tableList.get(0).getKeyValueTableName());
+    }
+
+    /**
+     * Tests Creation and deletion of KVTables and Streams +
+     * under different scopes
+     */
+    @Test
+    public void testDeleteKVtableStream() throws ExecutionException, InterruptedException {
```

Comment:
This test is out of scope for KVTs. What does deleting Streams have to do with KVTs? Please remove the whole test or move it to another class.

Reply:
Sure, I'll remove this test.